src/main/kotlin/de/docs_as_co/intellij/plugin/drawio/editor/BaseDrawioWebView.kt

```diff
@@ -132,5 +133,6 @@ abstract class BaseDrawioWebView(val lifetime: Lifetime) {
     }
 
     protected abstract fun handleEvent(event: IncomingMessage.Event)
+    protected abstract fun handleResponse(response: IncomingMessage.Response)
```

Comment:
This should not be necessary, as responses are directly associated with requests! You should rather use `.then` on the promise returned by the send method.

Reply:
done